app/controllers/game_controller.rb

```diff
@@ -17,15 +17,19 @@ def turn
                               session["current_player"]
     @coordinates = get_coordinates(params[:id]) #if id != nil or ""
     @symbol = @players.players[session["current_player"].to_i].symbol
+    @name = @players.players[session["current_player"].to_i].name
 
     @finished = play @board, @coordinates, @symbol
-    @players.next!
+    @players.next! unless @finished
     session["players"] = @players.players
     session["current_player"] = @players.current_player
 
     respond_to do |format|
       format.json do
-        render json: { board: @board, players: @players, finished: @finished }
+        render json: {
+                   :players => @players, :symbol => @symbol,
+                   :name => @name, :finished => @finished
+                     }
```

Comment:
Indent the right brace the same as the start of the line where the left brace is.